lib/iris/__init__.py

```diff
@@ -108,7 +108,7 @@ def callback(cube, field, filename):
 
 
 # Iris revision.
-__version__ = '1.7.4-DEV'
+__version__ = '1.7.4rc1'
```

Comment:
I'd still rather do a release candidate. I don't think that's at odds with the concept of a patch release at all.